.gitignore

```diff
@@ -19,6 +19,3 @@ src/ios/ios.c
 *.DS_Store*
 *-ios/
 __pycache__
-.tox
-venv
```

Comment:
actually I like having these two excluded. Only the tests should not be excluded but `.tox` and `venv` are good candidates